app/models/concerns/scripts/requires.rb

```diff
@@ -7,6 +7,7 @@ module Scripts::Requires
     has_many :requires, dependent: :destroy, foreign_key: :caller_id
     has_many :dependants, dependent: :destroy, class_name: 'Require'
     has_many :includes, through: :requires, source: :script
+    has_many :includes_libraries, through: :includes, source: :libraries
```

Comment:
¿Porque no solo `libraries`?

Reply:
porque el libraries ya está en otro concerns, el has_many libraries es el del accepts_nested_attributes y el includes_libraries, son las libraries de los scripts "incluídos", cuando hacer el "merge" de varios scripts, me entendés?

Reply:
Ahh claro, ya está, entonces igual le cambiaría el nombre, pero por esto:
```suggestion
    has_many :included_libraries, through: :includes, source: :libraries
```
Sino suena medio raro.

Reply:
oka